.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
-exclude: SBOM/
+exclude: ^(SBOM/)
```

Comment:
why changing this? it has no effect or?

Reply:
It just allows to extend it more easily in the future...